### ISSUE TITLE
Regression tests

### ISSUE THREAD

**Issue by sebastian-goeldi** [gdsfactory/kfactory#394]
Would be nice to have many of the tests also have regression tests like gdsfactory does.

**Comment by joamatab** [comment]
we can move that part to kfactory and kfactory can use it

**Comment by sebastian-goeldi** [comment]
yeah, absolutely. I was just being cheeky when I added help wanted ^^.

I think it would be great if we can add it, but it's not urgent at all
